src/main/io/spektrum_vtx_control.c

```diff
@@ -116,71 +128,87 @@ uint8_t convertSpektrumVtxPowerIndex(uint8_t sPower)
         .lowPowerDisarm = vtxSettingsConfig()->lowPowerDisarm,
     };
     vtxSettingsConfig_t newSettings = prevSettings;
 
-#ifdef USE_VTX_COMMON_FREQ_API
-    uint16_t freq = SpektrumVtxfrequencyTable[vtx.band][vtx.channel];
     if (vtxCommonDeviceRegistered()) {
+
+#ifdef USE_VTX_COMMON_FREQ_API
+        uint16_t freq = SpektrumVtxfrequencyTable[vtx.band][vtx.channel];
         if (prevSettings.freq != freq) {
-            newSettings.band = VTX_COMMON_BAND_USER;
+            newSettings.band    = VTX_COMMON_BAND_USER;
             newSettings.channel = vtx.channel;
-            newSettings.freq = freq;
+            newSettings.freq    = freq;
         }
-    }
 
 #else
-    // Convert to the internal Common Band index
-    uint8_t band    = spek2commonBand[vtx.band];
-    uint8_t channel = vtx.channel +1; // 0 based to 1 based
-    if (vtxCommonDeviceRegistered()) {
+        // Convert to the internal Common Band index
+        uint8_t band    = spek2commonBand[vtx.band];
+        uint8_t channel = vtx.channel +1; // 0 based to 1 based
         if ((prevSettings.band != band) || (prevSettings.channel != channel)) {
-            newSettings.band = band;
+            newSettings.band    = band;
             newSettings.channel = channel;
-            newSettings.freq = vtx58_Bandchan2Freq(band, channel);
+            newSettings.freq    = vtx58_Bandchan2Freq(band, channel);
         }
-    }
 #endif
 
-    // Seems to be no unified internal VTX API std for popwer levels/indexes, VTX device brand specific.
-    uint8_t power = convertSpektrumVtxPowerIndex(vtx.power);
-    if (vtxCommonDeviceRegistered()) {
+        // Seems to be no unified internal VTX API std for popwer levels/indexes, VTX device brand specific.
+        uint8_t power = convertSpektrumVtxPowerIndex(vtx.power);
         if (prevSettings.power != power) {
-            newSettings.power = power;
+            newSettings.power   = power;
         }
-    }
 
-    // Everyone seems to agree on what PIT ON/OFF means
-    uint8_t currentPitMode = 0;
-    if (vtxCommonDeviceRegistered()) {
+        // Everyone seems to agree on what PIT ON/OFF means
+        uint8_t currentPitMode = 0;
         vtxCommonGetPitMode(&currentPitMode);
         if (currentPitMode != vtx.pitMode) {
             vtxCommonSetPitMode(vtx.pitMode);
         }
     }
 
     if(memcmp(&prevSettings,&newSettings,sizeof(vtxSettingsConfig_t))) {
-        vtxSettingsConfigMutable()->band = newSettings.band;
+        vtxSettingsConfigMutable()->band    = newSettings.band;
         vtxSettingsConfigMutable()->channel = newSettings.channel;
-        vtxSettingsConfigMutable()->power = newSettings.power;
-        vtxSettingsConfigMutable()->freq = newSettings.freq;
+        vtxSettingsConfigMutable()->power   = newSettings.power;
+        vtxSettingsConfigMutable()->freq    = newSettings.freq;
+#ifdef USE_SAVE_CONFIG_NOTIFICATION
         saveConfigAndNotify();
+#else
+        saveConfig();
```

Comment:
What is the reason for not using the same functionality for storing the configuration that is used everywhere else?

Reply:
No need to do all the things ´saveConfigAndNotify()´ does, write EE , read EE, etc etc. We only need to do write EE. And in principal, such a simple function is missing and should be available.

Reply:
This is also a remainder from the T&E exercises when trying to figure out what might cause the instability seen in #4991. I could probably change it back now, but that would mean unnecessary EE-prom read beeing done.

Reply:
OK, removed my `saveConfig()` function, reverting to the common way of doing things.

Reply:
Thanks for changing this.
My worry about the introduction of `saveConfig()` was that it made it look like this was the save method that was to be used by default, and would probably have led to it being used in pull requests instead of `saveConfigAndNotify()`, which is the right thing to do in most cases, due to the fact that it includes validation of the values.
But since `saveConfig()` is gone now, this is moot.